virtio-devices/src/block.rs

```diff
@@ -104,6 +108,7 @@ struct BlockEpollHandler {
     counters: BlockCounters,
     queue_evt: EventFd,
     request_list: HashMap<u16, Request>,
+    rate_limiter: RateLimiter,
```

Comment:
I'd rather see an `Option` here, instead of having the default `RateLimiter` in case we don't use any.